src/validation/CampaignSchema.tsx

```diff
@@ -103,13 +114,16 @@ export const CampaignSchema = object().shape({
                 .max(60, "Max 60 Characters")
                 .required("Body is required"),
               targetUrlValid: boolean().isTrue(
-                "Target URL is either invalid or still validating, please check Ad for more details."
+                "Target URL is either empty, invalid, or still validating. Please check Ad for more details."
```

Comment:
This message is clumsy :).  It'd be nice if it said what the problem actually was.

Reply:
Formik hook lifecycle gave me a bit of trouble, but got it figured out.